.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "playbooks/roles/caddy"]
 	path = playbooks/roles/caddy
 	url = https://github.com/samdoran/ansible-role-caddy.git
+[submodule "playbooks/roles/authorized_keys"]
```

Comment:
Since you are changing this file, I remembered there is an issue https://github.com/ansible/ansibullbot/issues/1197, do we want to fix that (not necessarily in this PR)? If not, feel free to close.

Reply:
Yup, I will look into it.